src/docs/010_manual/50_Frequently_asked_Questions.adoc

```diff
@@ -193,3 +193,24 @@ An easy way to test you configuration is to run `xeyes` in WSL.
 
 NOTE:: Please make sure that you wsl is up-to-date bei executing `wsl --update` - this is not part of your regular windows update!
 ====
+
+=== Q: I get an error that gradle dependencies cannot be downloaded because the internet access is resticted by a proxy.
```

Comment:
I get an error that gradle dependencies cannot be downloaded because a proxy is restricting internet access.